en/2-deeds.md

```diff
@@ -122,11 +122,12 @@ The "Mintium" model goes further than the freemium model, as the services are fr
 
 ## 2.7. Cities
 
-The World of Meeds is a virtual world where people can buy Deeds, similar to real estate properties in virtual cities. These Deeds are only available in limited quantities and can exist only if someone redeems them with the minting points they accumulate by staking.
+The World of Meeds is a virtual world where people can buy Deeds, similar to real estate properties in virtual Cities. The Deeds are only available in limited quantities within a City and can exist only if someone redeems them with the minting points they accumulate by staking.
```

Comment:
The World of Meeds is a virtual world where people can buy Deeds, in proximity to real estate properties in virtual Cities. The Deeds are only available in short supplies within a City and can exist only if someone redeems them with the minting points they accumulate by staking.

Reply:
Few changes were made such as:
Changing SIMILAR to PROXIMITY and changing LIMITED QUANTITIES to SHORT SUPPLIES

Reply:
disagreed. Deeds are in shot supplies only on the first cities, but the total supply is huge.